CRM/Core/BAO/OptionGroup.php

```diff
@@ -67,7 +67,7 @@ public static function add(&$params, $ids = []) {
     if (empty($params['name']) && empty($params['id'])) {
       $params['name'] = CRM_Utils_String::titleToVar(strtolower($params['title']));
     }
-    elseif (!empty($params['name']) && strpos($params['name'], ' ')) {
+    elseif (!empty($params['name']) && strpos(($params['name'] ?? ''), ' ')) {
```

Comment:
This one's odd. If $params['name'] is null it wouldn't even reach that clause. Is it actually null here or is it something like $params['name'] is an object, or a nested array?

Reply:
I can't find which test would be doing this. If it's not empty it can't be null.

Reply:
ok have updated I think your right but not 100% sure now